sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -265,6 +276,28 @@ def _test_series(self, pyfunc, name, input_data=None):
             exec_times, _ = get_times(pyfunc, test_data, iter_number=self.iter_number)
             self.test_results.add(name, 'Reference', test_data.size, test_results=exec_times)
 
+    def _test_series_binary_operations(self, pyfunc, name, input_data=None):
+        np.random.seed(0)
+        hpat_func = sdc.jit(pyfunc)
+        for data_length in self.total_data_length[name]:
+
+            # TODO: replace with generic function to generate random sequence of floats
+            data1 = np.random.ranf(data_length)
+            data2 = np.random.ranf(data_length)
+            A = pd.Series(data1)
+            B = pd.Series(data2)
+
+            compile_results = calc_compilation(pyfunc, A, B, iter_number=self.iter_number)
+
+            # Warming up
+            hpat_func(A, B)
```

Comment:
This uses sdc.jit, but it'd be better to use do_jit function instead, as warming should use the same decorator as subsequent get_times.